docs/platforms/javascript/guides/nuxt/install/cli-import.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: --import CLI flag
+sidebar_order: 2
+description: "Learn about using the node `--import` CLI flag."
+---
+
+## Understanding the `--import` CLI Flag
+
+The [`--import` CLI flag](https://nodejs.org/api/cli.html#--importmodule) in Node allows you to preload modules before the application starts.
+This flag is crucial for ensuring proper server-side initialization, as Sentry needs to be initialized before the rest of the application runs.
```

Comment:
I'd make it clear that this is only crucial if users need to follow this guide (assuming users got to this page without reading "Installation Methods" carefully. 

```suggestion
If you cannot use the SDK's <PlatformLink to="/install/dynamic-import/">default dynamic import behaviour</PlatformLink>, setting this flag is crucial for ensuring proper server-side initialization, as Sentry needs to be initialized before the rest of the application runs.
```